bundles/io/org.eclipse.smarthome.io.http.core/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.eclipse.smarthome.io.http.core</name>
```

Comment:
We already have a `org.eclipse.smarthome.io.http` bundle, so this somehow duplicates it.
I think we could easily move the classes of this bundle to a package `org.eclipse.smarthome.io.http.servlet` in the `org.eclipse.smarthome.io.http` bundle and remove this one here.

Reply:
I messed up dot files (I have global git ignore to not commit them by default) and then I been adding them manually on other machine. Will fix.